### ISSUE TITLE
[WebToolsE2E][Aspire] Deploying Aspire-Samples/AspireWithJavaScript solution using 'azd up', the status of vue container app shows failed: ContainerCrashing, and angular's status shows degraded.

### ISSUE THREAD

**Issue by v-sherryfan** [Azure/azure-dev#3460]
**REGRESSION INFO:** Blocked by bug [#3421](https://github.com/Azure/azure-dev/issues/3421) on Aspire 8.0.0-preview.3.24105.21

**INSTALL STEPS**

1. Clean machine: Win11 x64 23h2 ENU
2. Install VS 17.10 Preview 2 [ includes Aspire 8.0.0-preview.4.24128.4 ]
3. Apply NuGet dotnet-tools Feeds
4. Install Docker Desktop for Windows
5. Install latest azd 
6. Install latest Node.js

**REPRO STEPS**

1. Clone aspire-samples repo from main branch
2. Checkout the `Preview.4` branch
3. Update sdk version to `8.0.300-preview.24101.10` in global.json
4. Update Aspire version to latest `8.0.0-preview.4.24128.4` in Directory.Packages.props
5. Open samples\AspireWithJavaScript.sln
7. Running 'npm install' under AspireWithJavaScript.Angular/AspireWithJavaScript.React/AspireWithJavaScript.Vue 
8. Build
9. Deploy using azd
    - Open the solution folder in a Command Prompt
    - Runing 'azd init', select 'Use code in the current directory' > Confirm and continue initializing my app, type a new environment name, press enter.
    - When running azd init choose to expose 'weatherapi' to the Internet
    - Run azd up to deploy to Azure
10. Open the resource group, click the 'vue'/'angular' container app.

**ACTUAL:**
![image](https://github.com/Azure/azure-dev/assets/71361488/3e4018da-097c-4767-8890-2ac6703f6621)

![image](https://github.com/Azure/azure-dev/assets/71361488/809b891f-7ae2-4ae6-b68e-0c2f8f1485bb)

**Note**

1. But click the 'react' container app withoout any errors.

**EXPECTED**
No issues

**Comment by rajeshkamal5050** [comment]
@v-sherryfan is this reproducible still? saw an update on - blocked by bug https://github.com/Azure/azure-dev/issues/3421 being closed.

**Comment by v-sherryfan** [comment]
@rajeshkamal5050 Yes, this issue still repro on Aspire 8.0 P4, blocked by bug https://github.com/Azure/azure-dev/issues/3421 on Aspire 8.0 P3.

**Comment by rajeshkamal5050** [comment]
@vhvb1989 is this reproducible with Preview.5 and our daily bits?

**Comment by v-sherryfan** [comment]
Still reproducing on 17.10 Preview 3 [Includes Aspire 8.0.0-preview.5.24201.12] + azd [1.9.0-beta.1-daily] using aspire-samples `main` branch.

**Comment by v-sherryfan** [comment]
This issue is not reproduced on `release/8.0` branch [Aspire 8.0.0-preview.7.24251.11] using VS 17.10 Preview 7 FB.
